tests/Integration/Integration.Tests/PurchaseOrderModuleTests/WhenUpdatingPurchaseOrdersOverbookFields.cs

```diff
@@ -25,18 +26,17 @@ public void SetUp()
                                 {
                                     OrderNumber = 600179,
                                     Overbook = "Y",
-                                    OverbookQty = 1
+                                    OverbookQty = 1,
+                                    CurrentlyUsingOverbookForm = true
```

Comment:
not necessarily before this PR gets merged, but we should refactor this ugliness at some point

Reply:
yeah we can have a chat about options, probably when fully implementing the update full-po stuff would be a good time